data_collection/gazette/spiders/sp/sp_itaporanga.py

```diff
@@ -0,0 +1,11 @@
+from datetime import date
+
+from gazette.spiders.base.instar import BaseInstarSpider
+
+
+class SpItaporangaSpider(BaseInstarSpider):
+    TERRITORY_ID = "3522802"
+    name = "sp_itaporanga"
+    allowed_domains = ["itaporanga.sp.gov.br"]
+    base_url = "https://www.itaporanga.sp.gov.br/portal/diario-oficial"
+    start_date = date(2009, 4, 16)
```

Comment:
```suggestion
    start_date = date(2011, 6, 7)
```
De acordo com a [revisão](https://github.com/okfn-brasil/querido-diario/pull/1281#issuecomment-2392431554) da revisão: 

> Só teve um caso que precisa mudar: Itaporanga-SP. Ao ordenar o .csv da coleta completa, aparece que a primeira data é em 2009, mas a segunda data é 2011. Isso é bastante estranho. Busquei o documento no site e parece ter sido cadastrado errado pela prefeitura, pq as datas no documento são outras. Então, pra esse raspador, a data inicial correta é 2011-06-07.